src/App.jsx

```diff
@@ -1,38 +1,147 @@
 import logo from "/logo.png";
+
 import "./App.css";
-import { makeShuffledDeck } from "./utils.jsx";
+
 import { useState } from "react";
 
+import { makeShuffledDeck } from "./utils.jsx";
+import UI from "./UI.jsx";
+import Card from "./Card.jsx";
+
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
 function App(props) {
   // Set default value of card deck to new shuffled deck
   const [cardDeck, setCardDeck] = useState(makeShuffledDeck());
   // currCards holds the cards from the current round
   const [currCards, setCurrCards] = useState([]);
+  // condition to check if game
+  const [inGame, setInGame] = useState(true);
+  const [inRound, setInRound] = useState(true);
```

Comment:
These variables are named a bit odd. What exactly does this represent? If it is, if the game or round has started or not, I would name it like so. `hasGameStarted`, `hasRoundStarted` or something along those lines. Usually booleans start with is/are/have/has etc.